flux_led/__main__.py

```diff
@@ -940,7 +947,29 @@ def setRgbw(self, r=None, g=None, b=None, w=None, persist=True,
                 msg.append(int(w))
             else:
                 msg.append(int(0))
-            msg.append(special)
+
+            if self.protocol == "LEDENET":
+                # LEDENET devices support two white outputs for cold and warm. We set
+                # the second one here - if we're only setting a single white value,
+                # we set the second output to be the same as the first
+                if w2 is not None:
```

Comment:
I was doing code archeology, but without having some other data captures it's a little bit difficult. Were you able to dig up any other data captures?

My impression is there's 3 protocols currently:
LEDENET_ORIGINAL (5 bytes)
unnamed (8-byte) -- 4 channel (RGB + W)
LEDENET (9-byte) -- 5 channel (RGB + W + W)

Both the 8 and 9 byte variants use a write mask, the difference is how many white channels they send.

Currently only 0x25, 0x27, and 0x35 will send the second white channel.

Please go back and check out the diff for https://github.com/Danielhiversen/flux_led/commit/fc169a33a824c18a6054923963cb941c1a164f49 if you look at the before version, you'll see what I mean, I think.